src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -2119,6 +2119,16 @@ internal override IEnumerable<ImmutableArray<byte>> GetInternalsVisibleToPublicK
             return (result != null) ? result.Keys : SpecializedCollections.EmptyEnumerable<ImmutableArray<byte>>();
         }
 
+        internal override IEnumerable<string> GetInternalsVisibleToAssemblyNames()
+        {
+            EnsureAttributesAreBound();
+
+            if (_lazyInternalsVisibleToMap == null)
+                return SpecializedCollections.EmptyEnumerable<string>();
+
+            return _lazyInternalsVisibleToMap.Keys;
```

Comment:
I'm not sure what ordering guarantees `Keys` provides. Do we care for determinism?

Reply:
I'll order the way we loop through them. I don't _think_ it should actually matter for this case, but better to be safe.